src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -6,17 +6,21 @@
 using System.Diagnostics;
 using System;
 using System.Collections;
-using System.IO;
 using System.Reflection;
 using System.Threading;
 using System.Text;
 using System.Runtime.InteropServices;
 using System.Globalization;
+using System.IO.Pipes;
+using System.Threading.Tasks;
 
 namespace System.ServiceProcess.Tests
 {
     public class TestService : ServiceBase
     {
+        private bool _disposed;
+        private Task serverStarted;
```

Comment:
>serverStarted; [](start = 21, length = 14)

nit: naming, start private instance fields with _ like the others

Reply:
Perhaps something like _waitClientConnect will be more meaningful (naming is always hard! :))

---
In reply to: [164004986](https://github.com/dotnet/corefx/pull/26524#discussion_r164004986) [](ancestors = 164004986)